### PR TITLE
don't require manual building of generation scripts

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by John-Colvin** [comment]
ping @9il

File: gen/setup.sh [hunk 1]
```diff
@@ -49,4 +49,6 @@ mv dub.json.part dub.json
 
 unset IFS
 
+dub build :configure
+dub build :splice
```
Comment (9il) [inline gen/setup.sh:53]: Is it new dub feature?

Comment (John-Colvin) [inline gen/setup.sh:53]: I don't think so... it's the normal way of explicitly building only a submodule.

**Comment by 9il** [comment]
LGTM

**Comment by 9il** [comment]
Do not forget to update readme

**Comment by John-Colvin** [comment]
done

**Pull request merged**